.kitchen.cloud.yml

```diff
@@ -15,8 +15,8 @@ driver_config:
 
 provisioner:
   name: chef_solo
-  data_bags_path: test/integration/default/data_bags
-  encrypted_data_bag_secret_key_path: test/integration/default/encrypted_data_bag_secret
+  #data_bags_path: test/integration/default/data_bags
```

Comment:
These cause `kitchen` to fail if they don't exist. Since the cookbook doesn't use data_bags, I have removed these lines.